docs-guides/_static/documentation_options.js

```diff
@@ -8,7 +7,7 @@ var DOCUMENTATION_OPTIONS = {
     LINK_SUFFIX: '.html',
     HAS_SOURCE: true,
     SOURCELINK_SUFFIX: '',
-    NAVIGATION_WITH_KEYS: true,
+    NAVIGATION_WITH_KEYS: false,
```

Comment:
How did this change

Reply:
I guess it is due to the upgrade of the dependencies ... 
I didn't manually change anything